internal/ad/download.go

```diff
@@ -330,3 +332,21 @@ func downloadRecursive(ctx context.Context, client *libsmbclient.Client, url, de
 	}
 	return nil
 }
+
+// findLocalGPTIni will look for a GPT.INI file in the given path (non-recursive).
+// To account for case differences in the filename/extension, we have to walk
+// the directory and check each entry.
+func findLocalGPTIni(path string) (string, error) {
+	entries, err := os.ReadDir(path)
+	if err != nil {
+		return "", fmt.Errorf(i18n.G("could not read directory %q: %w"), path, err)
+	}
+
+	for _, entry := range entries {
+		if strings.EqualFold(entry.Name(), "GPT.INI") {
```

Comment:
Let’s imagine that you are using a samba machine to distribute the GPO (so Linux), you have `GPT.INI` and `gpt.ini`, which one is going to win then? Shouldn’t we first always try the "canonical" one before fallbacking to case insensitive research?
Does the samba shared mount even supports this?